crates/load-test/src/tasks/v05.rs

```diff
@@ -41,7 +41,7 @@ pub async fn task_block_by_number(user: &mut GooseUser) -> TransactionResult {
 pub async fn task_block_by_hash(user: &mut GooseUser) -> TransactionResult {
     get_block_by_hash(
         user,
-        Felt::from_hex_str("0x58d8604f22510af5b120d1204ebf25292a79bfb09c4882c2e456abc2763d4a")
+        Felt::from_hex_str("0x02c3adae98c15fb062687afb51f8a950b4e59d996e303e073c098b2adc61003d")
```

Comment:
iiuc the snapshot here is ~~pretty~~ very small - right (<= 1k blocks)? https://alpha4.starknet.io/feeder_gateway/get_block?blockHash=0x02c3adae98c15fb062687afb51f8a950b4e59d996e303e073c098b2adc61003d

I'm now wondering if it makes sense to make the network a parameter in the tests, wdyt?

Reply:
I don't really see the value of supporting multiple networks here? We want a somewhat consistent way of benchmarking, and having different networks doesn't help with that.

Testnet blocks are also nothing like mainnet blocks; which are much heavier and are the actual concern imo.

Supporting multiple networks means we would have to add multiple known fixtures per method etc which sounds like a pain for no gain?